utils/utils.js

```diff
@@ -171,7 +167,56 @@ const volunteerForNewGiftees = async (user, count) => {
   await batch.commit()
 
   // return the result after the database stuff is complete
-  return { success: result }
+  return { success: "Please refresh the giftee list" }
+}
+
+/**
```

Comment:
Lovely doc!

Reply:
You want more of this for the future then?

Reply:
Where possible, sure! Don't need to overdo it or spend much time on it, but it's big help to know the types when we're not dealing with TypeScript.

Reply:
Ye, ill probally work on it while you folks are doing up teh frontend as its not something that would break anything.